src/about.html

```diff
@@ -8,6 +8,17 @@
   </head>
   <body>
     <h1>About AEG</h1>
+    <h2 style="font-family: Garamond, serif;">Introduction:</h2>
+    <p style="font-family: Garamond, serif;">
+    Alpha Existential Graphs evolved around 1896 by Charles Sanders Peirce which is a diagrammatic system of logic. In other words, this type of diagrammatic system has special symbols that are implemented in the graph and with the addition of ovals to present the negation, the graphs become more general to display the first-order logic with equality. 
```

Comment:
> with the addition of ovals to present the negation, the graphs become more general to display the first-order logic with equality.

This is completely false. Ovals introduce negation. First order logic with equality requires predicates and quantification over terms, neither of which we can do in Alpha Existential Graphs. These features are only added by Beta Existential Graphs with the introduction of term atoms and lines.